flash_attn/cute/mask.py

```diff
@@ -76,6 +76,12 @@ def apply_mask(
             causal_row_offset = (
                 1 + self.seqlen_k - n_block * self.n_block_size - self.seqlen_q - thr_col_offset
             )
+            c = 0
+            col_limit_transformed = 0
+            ncol: cute.Constexpr = 0
```

Comment:
this is the only one that feels kinda sketch..

Reply:
Hi @drisspg 

This should already been fixed with 4.2.1 release